DQM/L1TMonitorClient/python/L1TStage2EventInfoClient_cfi.py

```diff
@@ -93,7 +93,13 @@
                         SystemLabel = cms.string("Calo Layer2"),
                         HwValLabel = cms.string("Stage2CaloLayer2"),
                         SystemDisable  = cms.uint32(0),
-                        QualityTests = cms.VPSet()
+                        QualityTests = cms.VPSet(
+                            cms.PSet(
```

Comment:
Please move this to the uGT section (around line 337) since it is the uGT that should become red if there is a problem on the link.